src/main/java/uk/gov/hmcts/cmc/claimstore/events/claimantresponse/RejectOrganisationPaymentPlanEvent.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.events.claimantresponse;
```

Comment:
@rattur and this class - i dont think we need

Reply:
Yes we can remove